src/entrypoints/content/content.scss

```
@@ -23,22 +23,6 @@ body {
-
-  &[data-layout="nav"] iframe:first-child {
-    max-width: 450px;
-  }
-
-  &[data-frames="1"] iframe {
-    max-width: unset !important;
-  }
-
   &[data-message] .message {
 
   }
```

Stylelint (17.14.0)

[error] Empty block (block-no-empty)

(block-no-empty)